servers/annotations-api/src/test/mutation/highlights-create.integration.ts

```diff
@@ -201,11 +201,8 @@ describe('Highlights creation', () => {
 
     it('should mark the list item as updated and log the highlight mutation', async () => {
       const updateDate = new Date(2022, 3, 3);
-
```

Comment:
no idea why this is necessary to do it this way, but MULTIPLE runs and i didnt see the issue with this change.

Reply:
wow, ok great find!!